### PR TITLE
Feature/secondary button variant

### PR DESCRIPTION
**Pull request by FatimaGC**

### CODE REVIEW

**Comment by FatimaGC** [comment]
@kee-oth Do you think I'm good to merge this?

**Comment by kee-oth** [comment]
@FatimaGC 

I'm seeing some differences between the Primary and Secondary buttons:

1. Primary is rounded while secondary is squared off
2. Secondary "border" (it's actually an outline) makes the overall visual dimensions of the button larger. I recommend using the border property for this instead of outline. You can't make an outline rounded and border actually counts for the size of the button, which we'd want for this (otherwise, other elements will visually overlap with the button since the outline doesn't take up "physical" space, other elements act like it's not even there)
3. The hover state of the secondary button looks odd because of the outline not being rounded.

Primary
Screenshot 2023-05-02 at 7 31 04 PM

Secondary
Screenshot 2023-05-02 at 7 31 15 PM

Secondary hover
Screenshot 2023-05-02 at 7 36 57 PM

**Comment by kee-oth** [comment]
@FatimaGC There's also no shadow on secondary variant. Let's make sure that it's there for all the variants 👍

**Comment by FatimaGC** [comment]
@kee-oth I checked and this is how things are looking on my end:

Secondary
Screenshot 2023-05-03 at 7 39 50 PM

Secondary Hover 
Screenshot 2023-05-03 at 7 39 55 PM

Not sure why there's a discrepancy

**Comment by kee-oth** [comment]
@FatimaGC 

Looking better! But the buttons are different sizes, let's figure out what's going on there and fix it. Notice how the dimensions of the grid/box the same but in the Secondary button the border is on the _outside_ of the lines while on the Primary button the border is on the _inside_:

Primary
Screenshot 2023-05-04 at 9 43 29 PM

Secondary
Screenshot 2023-05-04 at 9 43 52 PM

**Comment by FatimaGC** [comment]
> @FatimaGC
> 
> Looking better! But the buttons are different sizes, let's figure out what's going on there and fix it. Notice how the dimensions of the grid/box the same but in the Secondary button the border is on the _outside_ of the lines while on the Primary button the border is on the _inside_:
> 
> Primary Screenshot 2023-05-04 at 9 43 29 PM
> 
> Secondary Screenshot 2023-05-04 at 9 43 52 PM

@kee-oth Is the solution to this to add `box-sizing: border-box` to `.button` class?

File: src/components/Button/styles.css [hunk 1]
```diff
@@ -16,20 +38,60 @@
...
+  border: 2px solid var(--buttonColor--hover);
+}
+
+.button--variant--SECONDARY {
+  --buttonStatus-disabled-font-color: var(--color-secondary-disabled-font-color);
+  --buttonStatus-disabled-outline: var(--color-primary-disabled-background);
+  background-color: var(--color-white);
+  color: var(--buttonColor--primary);
+  filter: drop-shadow(0 2px 4px #92a1ad);
+  border: 2px solid var(--buttonColor--primary);
```
Comment (kee-oth) [inline src/components/Button/styles.css]: Let's alphabetically sort this property 🔤

**Review by kee-oth — CHANGES_REQUESTED** [review]

**Comment by kee-oth** [comment]
@FatimaGC Looks good to merge after that one change 💪

**Pull request merged**